lblod/ember-rdfa-editor
# Migrate to pnpm

### Overview
This PR includes a migration from `npm` to `pnpm`, `pnpm` has the following advantages over `npm`:
- Faster dependency installation
- Option to use hard linking using the `file:` protocol: https://pnpm.io/cli/link#whats-the-difference-between-pnpm-link-and-using-the-file-protocol

##### connected issues and PRs:
N/A

### Setup
- Make sure you have `yarn@1.22.19 ` installed globally

### How to test/reproduce
- `pnpm install`
- `pnpm start` should still work as expected
- `pnpm lint:types` should yield no errors


### Challenges/uncertainties
- the `skipLibCheck` flag in the `tsconfig.json` file is enabled do to type issues with the `ember-intl` dependency.
